docs/API-standaarden.md

```diff
@@ -3,21 +3,37 @@ layout: landing-page
 
-Om doelmatig en efficiënt met API's te werken is het belangrijk om de manier waarop API's worden ontwikkeld, gedocumenteerd, getest en werken te standaardiseren. Er zijn een aantal manieren om dit te realiseren. Een daarvan is het kiezen voor een bepaalde architectuurstijl. Net als in de rest van de wereld geldt ook voor de Nederlandse overheid dat de voorkeur momenteel uitgaat naar API's die werken volgens de [REST-architectuurstijl](https://en.wikipedia.org/wiki/Representational_state_transfer) . Maar dan nog geldt dat er aanvullende ontwerpkeuzes moeten worden gemaakt. Daarom is afgesproken om overheidsbrede afspraken te maken over hoe de REST-stijl wordt toegepast bij het ontwerpen van API's. Deze afspraken zijn vastgelegd in de [REST-API Design Rules](https://www.forumstandaardisatie.nl/open-standaarden/rest-api-design-rules) standaard.
+Wil je, bijvoorbeeld als inkoper, weten welke API-standaarden aanbevolen of verplicht zijn voor een specifiek type applicatie? Neem dan een kijkje op [GEMMA online](https://gemmaonline.nl/index.php/API-standaarden) of raadpleeg de [Softwarecatalogus](https://www.softwarecatalogus.nl/pakketten) om te zien welke softwarepakketten een bepaalde standaard ondersteunen.
 
-Op de website [ developer.overheid.nl](https://developer.overheid.nl) is te zien dat er steeds meer overheids-API's beschikbaar komen die allerlei soorten functionaliteit aanbieden. Op [data.overheid.nl](https://data.overheid.nl/) worden (ook) API-specificaties gepubliceerd waarmee open data is op te vragen. Voor gemeenten zijn met name API's interessant waarmee elders vastgelegde brongegevens toegankelijk worden gemaakt. Gebruik daarvan kan bijvoorbeeld leiden tot hogere actualiteit en betrouwbaarheid van data en het onderhouden van lokale kopieën overbodig maken. 
+## Wat is een API-standaard
 
-De [GEMMA Architectuurvisie](https://www.gemmaonline.nl/index.php/Architectuurvisie) beschrijft hoe standaardisatie via het gebruik van API's cruciaal is om toe te kunnen werken naar een fundamenteel betere informatievoorziening voor gemeenten. Iets dat verder wordt uitgewerkt binnen de [thema-architectuur Common Ground](https://www.gemmaonline.nl/index.php/Thema-architectuur_Common_Ground) die de architectuur uitwerkt die nodig is om de doelen van Common Ground te realiseren. 
+Een Application Programming Interface (API) is een manier om twee of meer computerprogramma's met elkaar te laten communiceren. Het is een soort software-interface die een dienst aanbiedt aan andere stukjes software.
 
-Vanuit de VNG wordt op verschillende manieren gewerkt aan de ontwikkeling van gestandaardiseerde API's: 
+De API's werken volgens de [REST-architectuurstijl](https://en.wikipedia.org/wiki/Representational_state_transfer) en volgen de [REST-API Design Rules](https://www.forumstandaardisatie.nl/open-standaarden/rest-api-design-rules) standaard.
```

Comment:
Wellicht 'De API's' vervangen door 'De bij VNG Realisatie ontwikkelde API's'.

Reply:
Lijkt me overbodig. Deze context staat al in de eerste alinea.

Reply:
Ah ja, eens.